papas/graphtools/graphbuilder.py

```diff
@@ -39,13 +39,15 @@ def __init__(self, ids, edges):
 
         # build the subgraphs of connected nodes
         self.subgraphs = []
-        for subgraphlist in DAGFloodFill(self.nodes).blocks: # change to subgraphs
+        #sort option  below is needed for consistent orderings and is required for a match with papascpp
+        for subgraphlist in DAGFloodFill(self.nodes, sort_key).subgraphs:
             element_ids = [] 
             # NB the nodes that are found by FloodFill are the Nodes describing links between items
             # we want the ids of these nodes
             for node in subgraphlist:
                 element_ids.append(node.get_value())        
-            self.subgraphs.append(sorted(element_ids)) #newsort
+            self.subgraphs.append(element_ids) 
```

Comment:
DONE: I think you are right and it is safer to leave a sort here. I have made it a reverse sort to be consistent.